<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f4f4f4; /* Fundo claro */
    border: 1px solid #d3d3d3; /* Bordas sutis */
    border-radius: 10px; /* Bordas arredondadas */
}

QMenuBar {
    background-color: #ffffff; /* Fundo do menu */
    border-bottom: 1px solid #d3d3d3; /* Linha separadora */
    font-size: 14px;
}

QMenuBar::item {
    padding: 5px 15px;
    background-color: transparent;
}

QMenuBar::item:selected {
    background-color: #e0e0e0; /* Destaque ao passar o mouse */
}

QStatusBar {
    background-color: #ffffff;
    border-top: 1px solid #d3d3d3;
    font-size: 12px;
    color: #606060;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Jairo Tecnologia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Desenvolvimento de softwares</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Produto - (Fitas ou barcode)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Local de armazenamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProduto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLocalArmazenamento">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstoque">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #707070;
	color: rgb(161, 161, 161);
    border-radius: 10px;
    color: white;
    font-size: 16px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #5a52d4;
}</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelatorio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #707070;
	color: rgb(161, 161, 161);
    border-radius: 10px;
    color: white;
    font-size: 16px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #5a52d4;
}</string>
    </property>
    <property name="text">
     <string>Relatorio</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConfirmacao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descrição do item </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDescricao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota Fiscal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNf">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #c4c4c4;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
    background-color: #f5f5f5;
}
QLineEdit:focus {
    border: 2px solid #6c63ff;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Gestao de Fitas e Barcodes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tela de cadastro</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
